WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/Synchronizer/ProductInputTransformerTests.swift

```diff
@@ -116,6 +228,22 @@ class ProductInputTransformerTests: XCTestCase {
         XCTAssertEqual(update2.items.count, 0)
     }
 
+    // TODO: Failing test
```

Comment:
Failing test fixed with https://github.com/woocommerce/woocommerce-ios/pull/9173/commits/942d0ebbbf76af518fa409c99e77ac0a5ca6b66d . Remove the comment before merging.